### PR TITLE
🌐 Update Turkish translation for `docs/tr/docs/tutorial/first-steps.md`

### PR DESCRIPTION
**Pull request by hasansezertasan**
🌐 Update Turkish translation for `docs/tr/docs/tutorial/first-steps.md`

Updates outdated. Fixes mistranslation.

[Original File](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/tutorial/first-steps.md)

Discussion: #9193

### CODE REVIEW

File: docs/tr/docs/tutorial/first-steps.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+
+### Adım 2: Bir `FastAPI` "örneği" oluşturalım
+
+```Python hl_lines="3"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+Burada `app` değişkeni `FastAPI` sınıfının bir örneği olacaktır.
+
+Bu tüm API'yı oluşturmak için ana etkileşim noktası olacaktır.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:152]:
```suggestion
Bu, tüm API'yı oluşturmak için ana etkileşim noktası olacaktır.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+
+`uvicorn` komutunda atıfta bulunulan `app` ile aynıdır.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:154]:
```suggestion
Bu `app` değişkeni, `uvicorn` komutunda atıfta bulunulan değişkenin ta kendisidir.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+
+```console
+$ uvicorn main:my_awesome_api --reload
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+</div>
+
+### Adım 3: *Yol Operasyonu* Oluşturmak
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:184]:
```suggestion
### Adım 3: Bir *Yol Operasyonu* Oluşturalım
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+```
+
+... yol şöyle olur:
+
+```
+/items/foo
+```
+
+!!! info "Bilgi"
+    "Yol" genellikle "<abbr title="Endpoint: Bitim Noktası">endpoint</abbr>" veya "<abbr title="Route: Yönlendirme/Yön">route</abbr>" olarak adlandırılabilir.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:203]:
```suggestion
    "Yol" genellikle "<abbr title="Endpoint: Bitim Noktası">endpoint</abbr>" veya "<abbr title="Route: Yönlendirme/Yön">route</abbr>" olarak adlandırılır.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+
+Bir API oluştururken, "yol", "kaynakları" ile "endişeleri" ayırmanın ana yoludur.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:205]:
```suggestion
Bir API oluştururken, "yol", "kaynaklar" ile "endişeleri" ayırmanın ana yoludur.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 6]
```diff
@@ -0,0 +1,333 @@
...
+* `DELETE`
+
+...veya daha az kullanılan diğerleri:
+
+* `OPTIONS`
+* `HEAD`
+* `PATCH`
+* `TRACE`
+
+HTTP protokolünde, bu "metodlardan" birini (veya daha fazlasını) kullanarak her yol ile iletişim kurabilirsiniz.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:225]:
```suggestion
HTTP protokolünde, bu "metodlardan" birini (veya daha fazlasını) kullanarak her bir yol ile iletişim kurabilirsiniz.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 7]
```diff
@@ -0,0 +1,333 @@
...
+
+---
+
+API'lar oluştururkan, belirli bir amaca hizmet eden belirli HTTP methodlarını kullanırsınız.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:229]:
```suggestion
API'lar oluştururkan, belirli bir amaca hizmet eden belirli HTTP metodlarını kullanırsınız.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 8]
```diff
@@ -0,0 +1,333 @@
...
+
+Normalde kullanılan:
+
+* `POST`: veri oluşturmak.
+* `GET`: veri okumak.
+* `PUT`: veriyi güncellemek.
+* `DELETE`: veriyi silmek.
+
+Bu nedenle, OpenAPI'de HTTP methodlarından her birine "operasyon" denir.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:238]:
```suggestion
Bu nedenle, OpenAPI'de HTTP metodlarından her birine "operasyon" denir.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 9]
```diff
@@ -0,0 +1,333 @@
...
+
+Bizde onlara "**operasyonlar**" diyeceğiz.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:240]:
```suggestion
Biz de onları "**operasyonlar**" olarak adlandıracağız.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 10]
```diff
@@ -0,0 +1,333 @@
...
+
+#### Bir *Yol Operasyonu Dekoratörü* Tanımlayalım
+
+```Python hl_lines="6"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+`@app.get("/")` **FastAPI**'a hemen altındaki fonksiyonun aşağıdaki durumlardan sorumlu olduğunu söyler:
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:248]:
```suggestion
`@app.get("/")` dekoratörü, **FastAPI**'a hemen altındaki fonksiyonun aşağıdaki durumlardan sorumlu olduğunu söyler:
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 11]
```diff
@@ -0,0 +1,333 @@
...
+
+* <abbr title="Bir HTTP GET metodu"><code>get</code> operasyonu</abbr> ile
+* `/` yoluna gelen istekler
+
+!!! info "`@decorator` Bilgisi"
+    Python'da `@something` sözdizimi "<abbr title="Decorator">dekoratör</abbr>" olarak adlandırılır.
+
+    Dekoratörü bir fonksiyonun üzerine koyarsınız. Dekoratif bir şapka gibi (sanırım terim buradan geliyor).
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:256]:
```suggestion
    Dekoratörler, dekoratif bir şapka gibi (muhtemelen bu sebeple öyle adlandırılıyor), fonksiyonların üzerlerine yerleştirilirler.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 12]
```diff
@@ -0,0 +1,333 @@
...
+
+    Bir "dekoratör" hemen altında bulunan fonksiyonu alır ve o fonksiyon ile bazı işlemler gerçekleştirir.
+
+    Bizim durumumzda kullandığımız dekoratör **FastAPI**'a altındaki fonksiyonun `/` yoluna gelen `get` metodlu isteklerden sorumlu olduğunu söyler.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:260]:
```suggestion
    Bizim durumumuzda, kullandığımız dekoratör, **FastAPI**'a altındaki fonksiyonun `/` yoluna gelen `get` metodlu isteklerden sorumlu olduğunu söyler.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 13]
```diff
@@ -0,0 +1,333 @@
...
+
+    Bu bir **yol operasyon dekoratörü**.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:262]:
```suggestion
    Bu bir **yol operasyonu dekoratörüdür**.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 14]
```diff
@@ -0,0 +1,333 @@
...
+
+Ayrıca diğer operasyonları de kullanabilirsiniz:
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:264]:
```suggestion
Ayrıca diğer operasyonları da kullanabilirsiniz:
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 15]
```diff
@@ -0,0 +1,333 @@
...
+
+* `@app.post()`
+* `@app.put()`
+* `@app.delete()`
+
+Daha az kullanılanları da kullanabilirsiniz:
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:270]:
```suggestion
Daha egzotik olanları da tercih edebilirsiniz:
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 16]
```diff
@@ -0,0 +1,333 @@
...
+
+    **FastAPI** herhangi bir özel amacı veya anlamı olması konusunda ısrarcı olmaz.
+
+    Buradaki bilgiler bir gereklilik değil, bir kılavuz olarak sunulmaktadır.
+
+    Mesela GraphQL kullanırkan genelde tüm işlemleri yalnızca `POST` operasyonunu kullanarak gerçekleştirirsiniz.
+
+### Adım 4: **Yol Operasyonu Fonksiyonunu** Tanımlayın
+
+Aşağıdakiler bizim **yol operasyonu fonksiyonlarımızdır**:
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:288]:
```suggestion
Aşağıdaki, bizim **yol operasyonu fonksiyonumuzdur**:
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 17]
```diff
@@ -0,0 +1,333 @@
...
+
+* **yol**: `/`
+* **operasyon**: `get`
+* **fonksiyon**: "dekorar"ün (`@app.get("/")`) altındaki fonksiyondur.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:292]:
```suggestion
* **fonksiyon**: "dekoratör"ün (`@app.get("/")`'in) altındaki fonksiyondur.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 18]
```diff
@@ -0,0 +1,333 @@
...
+
+```Python hl_lines="7"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+Bu bir Python fonksiyonudur.
+
+Bir `GET` işlemi kullanarak "`/`" bağlantısına bir istek geldiğinde **FastAPI** tarafından çağrılır.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:300]:
```suggestion
Bu fonksiyon, ne zaman bir `GET` işlemi kullanılarak "`/`" bağlantısına bir istek geldiğinde **FastAPI** tarafından çağrılır.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 19]
```diff
@@ -0,0 +1,333 @@
...
+
+Bu durumda bir `async` fonksiyonudur.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:302]:
```suggestion
Bu durumda bu fonksiyon bir `async` fonksiyondur.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 20]
```diff
@@ -0,0 +1,333 @@
...
+
+---
+
+Bunu `async def` yerine normal bir fonksiyon olarakta tanımlayabilirsiniz.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:306]:
```suggestion
Bu fonksiyonu `async def` yerine normal bir fonksiyon olarak da tanımlayabilirsiniz.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 21]
```diff
@@ -0,0 +1,333 @@
...
+!!! note "Not"
+    Eğer farkı bilmiyorsanız, [Async: *"Aceleniz mi var?"*](../async.md#in-a-hurry){.internal-link target=_blank} sayfasını kontrol edebilirsiniz.
+
+### Adım 5: İçeriği Geri Döndürün
+
+```Python hl_lines="8"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+Bir `dict`, `list`veya `str`, `int` gibi tekil değerler döndürebilirsiniz.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:321]:
```suggestion
Bir `dict`, `list` veya `str`, `int` gibi tekil değerler döndürebilirsiniz.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 22]
```diff
@@ -0,0 +1,333 @@
...
+
+Ayrıca, Pydantic modellerini de döndürebilirsiniz (bununla ilgili daha sonra ayrıntılı bilgi göreceksiniz).
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:323]:
```suggestion
Ayrıca, Pydantic modelleri de döndürebilirsiniz (bu konu ileriki aşamalarda irdelenecektir).
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 23]
```diff
@@ -0,0 +1,333 @@
...
+
+Otomatik olarak JSON'a dönüştürülecek (ORM'ler vb. dahil) başka birçok nesne ve model vardır. En beğendiklerinizi kullanmayı deneyin, yüksek ihtimalle destekleniyordur.
+
+## Özet
+
+* `FastAPI`'yı projemize dahil ettik.
+* Bir `app` örneği oluşturduk.
+* **yol operasyonu dekoratörü** (`@app.get("/")` gibi) yazdık.
+* **yol operasyonu fonksiyonu** (`def root(): ...` gibi) yazdık.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:332]:
```suggestion
* Bir **yol operasyonu dekoratörü** (`@app.get("/")` gibi) oluşturduk.
* Bir **yol operasyonu fonksiyonu** (`def root(): ...` gibi) oluşturduk.
```

**Review by emrhnsyts — CHANGES_REQUESTED** [review]
Gerekli değişiklikler uygulandıktan sonra son olarak tekrar gözden geçireceğim. Ellerinize sağlık.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 1562798cc363e99e74577dfaa777d11d165f7ac1 at: https://995b1286.fastapitiangolo.pages.dev

File: docs/tr/docs/tutorial/first-steps.md (hunk 18 above)
Comment (hasansezertasan) [reply docs/tr/docs/tutorial/first-steps.md:300]: Önerin devrik duruyor, tekrar edebilir misin?

**Comment by tiangolo** [comment]
📝 Docs preview for commit 33345a10b93f22f9425812a27fc5b7db60e65827 at: https://dcf44357.fastapitiangolo.pages.dev

File: docs/tr/docs/tutorial/first-steps.md [hunk 24]
```diff
@@ -0,0 +1,333 @@
...
+
+    Tüm <a href="https://www.starlette.io/" class="external-link" target="_blank">Starlette</a> fonksiyonlarını `FastAPI` ile kullanabilirsiniz.
+
+### Adım 2: Bir `FastAPI` "örneği" oluşturalım
+
+```Python hl_lines="3"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+Burada `app` değişkeni `FastAPI` sınıfının bir örneği olacaktır.
```
[the remark this answers is not in the document]
Comment (hasansezertasan) [reply docs/tr/docs/tutorial/first-steps.md:150]: Daha önce bir nesnenin türevi  yani "instance"'ı için "örnek" kullanımı var diye hatırlıyorum.

**Comment by tiangolo** [comment]
📝 Docs preview for commit bba4cb908f498828e6f5c9f0011636d348e1d48a at: https://aab9f6d7.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 57f0bd8767546d08a913ceb48f0cfda567494d0f at: https://f21b0ecf.fastapitiangolo.pages.dev

File: docs/tr/docs/tutorial/first-steps.md [hunk 25]
```diff
@@ -0,0 +1,333 @@
...
+
+#### "Şema"
+
+"Şema", bir şeyin tanımı veya açıklamasıdır. Geliştirilen koddan ziyade soyut bir açıklamadır.
+
+#### API "Şeması"
+
+Bu durumda, <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a>, API şemasını nasıl tanımlayacağınızı belirten bir şartnamedir.
+
+Bu şema tanımı, API yollarınızla birlikte yollarınızın aldığı olası parametreleri gibi tanımlamaları içerir.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:80]:
```suggestion
Bu şema tanımı, API yollarınızla birlikte yollarınızın aldığı olası parametreler gibi tanımlamaları içerir.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 26]
```diff
@@ -0,0 +1,333 @@
...
+                        "description": "Successful Response",
+                        "content": {
+                            "application/json": {
+
+
+
+...
+```
+
+#### OpenAPI Ne İçindir?
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:121]:
```suggestion
#### OpenAPI Ne İşe Yarar?
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 27]
```diff
@@ -0,0 +1,333 @@
...
+... yol şöyle olur:
+
+```
+/items/foo
+```
+
+!!! info "Bilgi"
+    "Yol" genellikle "<abbr title="Endpoint: Bitim Noktası">endpoint</abbr>" veya "<abbr title="Route: Yönlendirme/Yön">route</abbr>" olarak adlandırılır.
+
+Bir API oluştururken, "yol", "kaynaklar" ile "endişeleri" ayırmanın ana yoludur.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:205]:
```suggestion
Bir API oluştururken, "yol", "kaynaklar" ile "endişeleri" ayırmanın ana yöntemidir.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 28]
```diff
@@ -0,0 +1,333 @@
...
+* `OPTIONS`
+* `HEAD`
+* `PATCH`
+* `TRACE`
+
+HTTP protokolünde, bu "metodlardan" birini (veya daha fazlasını) kullanarak her bir yol ile iletişim kurabilirsiniz.
+
+---
+
+API'lar oluştururkan, belirli bir amaca hizmet eden belirli HTTP metodlarını kullanırsınız.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:229]:
```suggestion
API oluştururkan, belirli bir amaca hizmet eden belirli HTTP metodlarını kullanırsınız.
```

File: docs/tr/docs/tutorial/first-steps.md [hunk 29]
```diff
@@ -0,0 +1,333 @@
...
+
+`@app.get("/")` dekoratörü, **FastAPI**'a hemen altındaki fonksiyonun aşağıdaki durumlardan sorumlu olduğunu söyler:
+
+* <abbr title="Bir HTTP GET metodu"><code>get</code> operasyonu</abbr> ile
+* `/` yoluna gelen istekler
+
+!!! info "`@decorator` Bilgisi"
+    Python'da `@something` sözdizimi "<abbr title="Decorator">dekoratör</abbr>" olarak adlandırılır.
+
+    Dekoratörler, dekoratif bir şapka gibi (sanırım terim buradan geliyor), fonksiyonların üzerlerine yerleştirilirler.
```
Comment (emrhnsyts) [inline docs/tr/docs/tutorial/first-steps.md:256]:
```suggestion
    Dekoratörler, dekoratif bir şapka gibi (sanırım terim buradan geliyor) fonksiyonların üzerlerine yerleştirilirler.
```

**Review by emrhnsyts — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit b9c3133461f961bd1e1afbc44263faaf4552aad8 at: https://1bea8536.fastapitiangolo.pages.dev

**Review by emrhnsyts — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit eac59cc90164498516dd5c2873919d9a62126a5d at: https://c914899d.fastapitiangolo.pages.dev

**Comment by hasansezertasan** [comment]
Thank you @emrhnsyts for the review!

**Comment by alejsdev** [comment]
Nice, thank you! @hasansezertasan 🥳🎉 

And thanks for the review @emrhnsyts 🔍🧐

**Pull request merged**